rllib/algorithms/apex_ddpg/apex_ddpg.py

```diff
@@ -151,4 +152,16 @@ def setup(self, config: AlgorithmConfig):
     @override(DDPG)
     def training_step(self) -> ResultDict:
         """Use APEX-DQN's training iteration function."""
+        if log_once("apex-ddpg-deprecation-warning"):
+            deprecation_warning(
+                old="rllib/algorithms/apex_ddpg/",
+                new="rllib_contrib/apex_ddpg/",
+                help=(
+                    "This algorithm will be "
```

Comment:
change to "This algorithm will be removed from rllib"
" It is being moved to the ray/rllib_contrib directory"